pallets/marketplace/src/lib.rs

```diff
@@ -330,7 +334,36 @@ pub mod pallet {
                     pallet_acurast::Error::RevokedCertificate => true,
                     _ => false,
                 },
-                _ => false,
+                Error::CapacityNotFound => true,
+
+                Error::CalculationOverflow => false,
+                Error::JobRegistrationZeroDuration => false,
+                Error::JobRegistrationScheduleExceedsMaximumExecutions => false,
+                Error::JobRegistrationScheduleContainsZeroExecutions => false,
+                Error::JobRegistrationDurationExceedsInterval => false,
+                Error::JobRegistrationStartInPast => false,
+                Error::JobRegistrationEndBeforeStart => false,
+                Error::JobRegistrationZeroSlots => false,
+                Error::JobStatusNotFound => false,
+                Error::JobRegistrationUnmodifiable => false,
+                Error::CannotAcknowledgeWhenNotMatched => false,
+                Error::CannotReportWhenNotAcknowledged => false,
+                Error::AdvertisementNotFound => false,
+                Error::AdvertisementPricingNotFound => false,
+                Error::TooManyAllowedConsumers => false,
+                Error::TooFewAllowedConsumers => false,
+                Error::TooManySlots => false,
+                Error::CannotDeleteAdvertisementWhileMatched => false,
+                Error::FailedToPay => false,
+                Error::AssetNotAllowedByBarrier => false,
+                Error::ReportFromUnassignedSource => false,
+                Error::MoreReportsThanExpected => false,
+                Error::ReportOutsideSchedule => false,
+                Error::ReputationNotFound => false,
+                Error::JobNotAssigned => false,
+                Error::JobCannotBeFinalized => false,
+
+                Error::__Ignore(_, _) => false,
```

Comment:
We should prefer this:
```suggestion
                _ => false,
```

Only the ones returning `true` should be listed